lib/open-amp/vendor/nxp_addr_translation.h

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2025 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+static const struct addr_trans_region regions[] = {
+	{
+		.addr = 0x1bef0000,
+		.remote_addr = 0x8fef0000,
+		.size = 0x10000000
```

Comment:
minor nit. Would be nice with comment informing from where those values originates.

Reply:
added comments -see https://github.com/zephyrproject-rtos/zephyr/pull/84170/files#diff-8b96f0f54f5cf721c371626c99fde50de1f3fdf0ca53445435d33144796c52c7R10